spec/index.html

```diff
@@ -659,8 +687,7 @@ <h3>IRIs</h3>
           “<code>%3f</code>”)</li>
         <li>Punycode-encoding of Internationalized Domain Names
           in IRIs</li>
-        <li>IRIs that are not in Unicode Normalization
-          Form C [[NFC]]</li>
+        <li>IRIs that are not in Unicode <a class="lint-ignore">Normalization Form C</a> [[UNICODE]]</li>
```

Comment:
Might immediately refer to the specific annex here.

```suggestion
        <li>IRIs that are not in Unicode <a class="lint-ignore">Normalization Form C</a> [[UAX15]]</li>
```